client/e2e/support/user-list.po.ts

```diff
@@ -0,0 +1,40 @@
+export class UserListPage {
+  navigateTo() {
+    return cy.visit('/users');
+  }
+
+  getUrl() {
+    return cy.url();
+  }
+
+  getUserTitle() {
+    return cy.get('.user-list-title');
+  }
+
+  getUserCards() {
+    return cy.get('.user-cards-container app-user-card');
+  }
+
+  getUserListItems() {
+    // return element(by.className('user-nav-list')).all(by.className('user-list-item'));
+    return cy.get('.user-nav-list .user-list-item');
+  }
+
+  clickViewProfile(card: Cypress.Chainable<JQuery<HTMLElement>>) {
+    // return card.element(by.buttonText('VIEW PROFILE')).click();
+    return card.contains('button', 'VIEW PROFILE').click();
+  }
+
+  changeView(viewType: 'card' | 'list') {
+    return cy.get(`#view-type-radio .mat-radio-button[value="${viewType}"]`).click();
+  }
+
+  selectRole(value: string) {
+    // This is messy. I'm not sure that Cypress is working well for us here. Might want to find a better way.
+    return cy.get('#user-role-select').click().get(`mat-option[value="${value}"]`).click();
```

Comment:
It is a bit messy, but it does accurately reflect the steps required to set a drop-down. If there's an easier/cleaner way I'm good with that, but this seems plausible as is.

That said, we could improve the formatting and include some explanation, e.g.,

```javascript
    return 
        // Find and click the drop down
        cy.get('#user-role-select').click()
          // Select and click the desired value from the resulting menu
          .get(`mat-option[value="${value}"]`).click();
```